SI/Sem.02/Pract.02/Task01.cpp

```diff
@@ -0,0 +1,17 @@
+#include <iostream>
+
+int main() {
+	int year = 0;
```

Comment:
Is it me or do we have x2 tabulations ?

Reply:
It's not just you, I can see them as well. it is definitely not on purpose. There should be only one tabulation. I don't know why that has happened, but in the source file the tabs are single. I've attached a screenshot of the source file that I've uploaded here.
![Task01Screenshot](https://user-images.githubusercontent.com/44443630/195726128-d988bff2-4c30-45a0-843e-a95abeab6e31.png)

Reply:
I resolved the issue. It's just GitHub that displays the single tab so big that it looks like 2 tabs. But in reality, when copying the code from the file and pasting it in VS the tab is only 1. In conclusion it's just a visual "bug".